### PR TITLE
Create a custom image for Kaniko

### PR DESCRIPTION
**Pull request by diego-ojeda-binbash**
## what
* Build a custom image for Kaniko

## why
* We quickly put together this one for a client because we needed to troubleshoot a build pipeline and the official image didn't allow us to add the binaries we needed

### CODE REVIEW

File: kaniko/Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+help:
+	@echo 'Available Commands:'
+	@egrep '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":"}; { if ($$3 == "") { printf " - \033[36m%-18s\033[0m %s\n", $$1, $$2 } else { printf " - \033[36m%-18s\033[0m %s\n", $$2, $$3 }}'
+
+#==============================================================#
+# DOCKER                                                       #
+#==============================================================#
+build: ## build docker image
+	docker build -t public.ecr.aws/n0c8p2u1/kaniko:latest --build-arg KANIKO_VERSION='1.9.0' .
```
Comment (diego-ojeda-binbash) [inline kaniko/Makefile:13]: Notice we are pushing this to our public ECR repo which can be accessed from here: https://gallery.ecr.aws/n0c8p2u1/

**Review by exequielrafaela — APPROVED** [review]

**Review by exequielrafaela — COMMENTED** [review]
@diego-ojeda-binbash OJ not sure why CI its failing https://app.circleci.com/pipelines/github/binbashar/le-docker-images/366/workflows/3e9d4fe9-5f0a-49cc-98c8-019a38d5945c/jobs/578

**Comment by exequielrafaela** [comment]
@juanmatias so as discuss the pending job here it's associated with the corresponding makefiles to build and push this image to dockerhub binbash account instead of the AWS public ECR to keep the standard approach. Thanks in advance for your collaboration 💪🏼

File: kaniko/Makefile (hunk 1 above)
Comment (diego-ojeda-binbash) [reply kaniko/Makefile]: FYI @juanmatias just took care of making Kaniko part of our Docker Hub hosted images. Now, we need to make a note to switch to that new image which we are using on a client.
+ @fgauchat-binbash  ☝️ 🙏

**Pull request merged**